pallas-network/src/multiplexer.rs

```diff
@@ -63,112 +65,126 @@ pub struct Segment {
 }
 
 pub enum Bearer {
-    Tcp(TcpStream),
+    Tcp(tcp::TcpStream),
 
     #[cfg(unix)]
-    Unix(UnixStream),
-
-    #[cfg(windows)]
-    NamedPipe(NamedPipeClient),
+    Unix(unix::UnixStream),
+    // #[cfg(windows)]
```

Comment:
dropping windows support?

Reply:
only temporary, I'll add it back once I find a way to test. It will be before next release but in a different branch.